cocos/dragon-bones/ArmatureDisplay.ts

```diff
@@ -142,12 +142,14 @@ interface BoneIndex extends Number {
  * Armature Display has a reference to a DragonBonesAsset and stores the state for ArmatureDisplay instance,
  * which consists of the current pose's bone SRT, slot colors, and which slot attachments are visible. <br/>
  * Multiple Armature Display can use the same DragonBonesAsset which includes all animations, skins, and attachments. <br/>
+ * Cocos Creator supports DragonBones version to 5.6.300.
```

Comment:
All versions lower than 5.6.300 are supported?

Reply:
> All versions below 5.6.300 are supported?

Depends on the compatibility of the DragonBones runtime library itself.
Most low version resources should be fine.